pom.xml

```diff
@@ -142,6 +142,7 @@
         <module>presto-testing</module>
         <module>presto-testing-kafka</module>
         <module>presto-testing-server-launcher</module>
+        <module>presto-testng-services</module>
```

Comment:
can we use presto-testing for this?

Reply:
No, I do not want to use `presto-testing`(white box testing)  in product tests (black box testing).

BTW product tests already depends on presto-main which is bad.